distrax/_src/distributions/gamma_test.py

```diff
@@ -0,0 +1,215 @@
+# Copyright 2021 DeepMind Technologies Limited. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for `gamma.py`."""
+
+from absl.testing import absltest
+from absl.testing import parameterized
+
+import chex
+from distrax._src.distributions import gamma
+from distrax._src.utils import equivalence
+import jax.numpy as jnp
+import numpy as np
+
+RTOL = 2e-2
+
+
+class GammaTest(equivalence.EquivalenceTest, parameterized.TestCase):
+
+  def setUp(self):
+    # pylint: disable=too-many-function-args
+    super().setUp(gamma.Gamma)
+    self.assertion_fn = lambda x, y: np.testing.assert_allclose(x, y, rtol=RTOL)
+
```

Comment:
Could you also add tests for the properties `concentration` and `rate`? Just to check that the broadcasting is working as intended.

Reply:
@franrruiz I am not sure what you mean here precisely?

Reply:
I just meant to add a test where `concentration` has shape (say) `[4, 2]` and `rate` is `[2]`. We would test that `distribution.concentration` coincides with the input value, and `distribution.rate` is `[4, 2]` and has been broadcasted properly. (And the same for the reverse, where the broadcasting happens on `concentration`). This might seem like a simple thing to test but it's helpful mainly for maintenance.